### DISCUSSION TITLE
How to convert long pointer(wrapped char*) to string(java.lang.String)

### DISCUSSION THREAD

**Discussion by xtexx** [LWJGL/lwjgl3#696] [Q&A] [answered]
As the title, in `BGFXFatalCallbackI`, argument `msg` wrapped `char*` as `long`

**Comment by Spasi** [comment, marked as answer]
It's a null-terminated string, so `MemoryUtil.memASCII(_str)` will do the job.

**Reply by xtexx** [reply]
thanks
